### PR TITLE
Adding transformato

### PR DESCRIPTION
**Pull request by JohannesKarwou**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by JohannesKarwou** [comment]
@wiederm I included you as a recipe maintainer, can you leave a comment in this PR to confirm that it is okay for you to be listed as a maintainer?

**Comment by wiederm** [comment]
Thanks @JohannesKarwou for the good work! Super happy to see this happening and super happy to be listed as maintainer!

**Comment by JohannesKarwou** [comment]
@conda-forge/help-python, ready for review!

File: recipes/transformato/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+package:
+  name: transformato
+  version: 0.3
+
+source:
+  url: https://github.com/wiederm/transformato/archive/refs/tags/v0.3.tar.gz
+  sha256: 00a8b8ac33a49a179035fc2ace86c382bad0f94363d95a7aa6a0f0157c181156
+  
+build:
+  skip: true  # [win or osx]
```
Comment (dopplershift) [inline recipes/transformato/meta.yaml:12]: This looks like it's pure Python and could be noarch. What's driving the need to skip win/osx? If it's missing dependencies, we can definitely make this noarch and ignore the resulting CI failures.

Comment (JohannesKarwou) [reply recipes/transformato/meta.yaml]: Thanks for the fast response! Yes it's pure python, I was just confused by the failing CI tests on win/osx I updated the recipe and added `noarch: python`

**Review by dopplershift — APPROVED** [review]

**Pull request merged**